### DISCUSSION TITLE
Could NOT find GTest while building  opentelemetry-cpp in window 10 using cmake

### DISCUSSION THREAD

**Discussion by DCMinds** [open-telemetry/opentelemetry-cpp#1644] [General]
Hi All, 
I am building opentelemetry-cpp by fowwling the steps mentioned in https://github.com/open-telemetry/opentelemetry-cpp/blob/main/INSTALL.md#building-shared-libs-for-windows
 but getting the below error
 
  C:\WS\Source\opentelemetry-cpp\build> cmake ..
-- Selecting Windows SDK version 10.0.19041.0 to target Windows 10.0.19042.
-- Building for architecture ARCH=x64
Trying to use local nlohmann::json from submodule
-- Building for architecture ARCH=x64
Trying to use local nlohmann::json from submodule
-- Using the single-header code from C:/WS/Source/opentelemetry-cpp/third_party/nlohmann-json/single_include/
CMake Error at C:/Program Files/CMake/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find GTest (missing: GTEST_LIBRARY GTEST_INCLUDE_DIR
  GTEST_MAIN_LIBRARY)
Call Stack (most recent call first):
  C:/Program Files/CMake/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  C:/Program Files/CMake/share/cmake-3.24/Modules/FindGTest.cmake:270 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  CMakeLists.txt:375 (find_package)

-- Configuring incomplete, errors occurred!
See also "C:/WS/Source/opentelemetry-cpp/build/CMakeFiles/CMakeOutput.log".
See also "C:/WS/Source/opentelemetry-cpp/build/CMakeFiles/CMakeError.log".

I am stuck in this for long time

**Comment by owent** [comment]
opentelemetry-cpp has a lot of dependencies. You should install dependency packages before building opentelemetry-cpp.
The dependencies of components in opentelemetry-cpp can be find at https://github.com/open-telemetry/opentelemetry-cpp/blob/main/docs/dependencies.md .

The easiest way to install these packages is using a package manager, please see https://github.com/open-telemetry/opentelemetry-cpp/blob/main/INSTALL.md#using-package-managers or use `apt/yum/dnf/pacman/brew` or other package manager on your system.
